comment/api/serializers.py

```diff
@@ -97,6 +98,7 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         user = kwargs['context']['request'].user
+        self.email_service = None
```

Comment:
I can't seem to find if this initialization is used at a later stage in this method.

Reply:
It's used here https://github.com/Radi85/Comment/pull/127/files/e81dd3dd749bb16afccea6f4d139ac039c6622cb#diff-580565eef493475112cb9daed5cc31fd758b926848bc1ba626bab78ea81acc96R130-R134

Reply:
Yes, I saw it. My point was, initializing it to `None` is not used anywhere in this method.

Reply:
IMHO is a good practice to initialize an external service as an instance variable rather than using it as a scope varaible and to make an instance variable you should initialize it in the __init__ method otherwise lintpy will complain when using `self.var` inside a method other than __init__ before initialize it in it.

Reply:
okay okay I didn't see that light colored line in between that indicates a big chunk of code in between hasn't changed. My mistake, sorry to bother you. I agree with the initialization practice.